pkg/plugin/v3/scaffolds/init.go

```diff
@@ -101,11 +103,18 @@ func (s *initScaffolder) scaffold() error {
 		&templates.GitIgnore{},
 		&rbac.AuthProxyRole{},
 		&rbac.AuthProxyRoleBinding{},
-		&kdefault.AuthProxyPatch{},
+		&kdefault.AuthProxyPatch{
+			ComponentConfig: s.componentConfig,
+		},
```

Comment:
I feel like being a long nlist of templates, single lines for each are better unless really long.
```suggestion
		&kdefault.AuthProxyPatch{ComponentConfig: s.componentConfig},
```